custom_components/easee/controller.py

```diff
@@ -267,9 +272,9 @@ async def refresh_equalizers_state(self, now=None):
         for equalizer_data in self.equalizers_data:
             equalizer_data.state = await equalizer_data.equalizer.get_state()
             if equalizer_data.state["isOnline"]:
-                equalizer_data.state["isOnline"] = "ONLINE"
+                equalizer_data.state["isOnline"] = "eq_online"
```

Comment:
Const exists

Reply:
Result of a string replace that ran away...